gdx/src/com/badlogic/gdx/graphics/g2d/Sprite.java

```diff
@@ -529,7 +529,17 @@ public float getX () {
 	public float getY () {
 		return y;
 	}
-
+	
+	/** @return the center x of the sprite, not accounting for scale. */
+	public float getCenterX(){
+		return getX() + width / 2;
+	}
+	
+	/** @return the center y of the sprite, not accounting for scale. */
+	public float getCenterY(){
+		return getY() + height / 2;
+	}
+	

```

Comment:
I don't believe that the method call would add overhead these days and I call this premature optimization. If you feel otherwise use x and y.

Improving code clarity is always a priority.